stable/elasticsearch/templates/configmap.yaml

```diff
@@ -40,22 +42,22 @@ data:
         ping.unicast.hosts: ${DISCOVERY_SERVICE:}
         minimum_master_nodes: ${MINIMUM_MASTER_NODES:2}
 
-{{- if .Values.cluster.xpackEnable }}
+  {{- if .Values.cluster.xpackEnable }}
```

Comment:
helm templates are more readable if there is no indentation.

Reply:
I think this is more readable because this indentation make inner and outer if statements be easier to distinguish

Reply:
OK for this!